src/controller/genericmidi.cxx

```diff
@@ -623,28 +398,30 @@ void GenericMIDI::launchScene( int scene )
     
     if ( b->action == GRID_LAUNCH_SCENE )
     {
-      for( int i = 0; i < 5; i++ )
```

Comment:
@harryhaaren i dont understand what you mean by 1, but every scene gets a send on or off when you click one button. The off-messages are written. 

If this is such a big issue for you, simply dont merge it. the previous version was pretty bad because only the first five scene launch buttons were update but only if their midi messages are numbered without gaps. my version works for me but its only a little change. you can test it, or not. i am a little tired of discussing this time and time again, sorry! you have the patch, feel free to do anything you want with it, until it may be in the master i will patch it to my local version on my own. 
Again: big sorry. this isnt personal or something like this, its just about this topic. i dont know what i should say what i didnt said before and obviously you are not happy with this solution, but i am. maybe we dont get to an common point, so i will simply leave this as it is.